### ISSUE TITLE
Career Mode - Rescue mission: kerbalnaut being rescued goes immediately to refuses to work

### ISSUE THREAD

**Issue by brocksamson** [BobPalmer/USI-LS#107]
First, love your mods, played kerbal stock clear up to extensive exploration of Jool and wanted something more permanant and realistic. Your mods fit the bill precisely :+1:

This is my first bug report for a KSP mod, so let me know what additional info I need to provide.

I just started a new career mode in kerbal 1.1. Most of my mods are USI (life support, kolonization, exploration, ftt, art).

I'm at day 1, hour 2 in my playthrough and I just accepted my first rescue mission. When I get within 2 km of the stranded kerbalnaut I get an immediate [name] refuses to work message and am unable to eva the kerbal to my rescue ship. I haven't unlocked any way to take supplies to him yet, so I'm pretty much stuck. Is this intentional? I rescued many kerbals in 1.05 with the previous version of USI and this never occurred.

Is there a recommended way to rescue kerbals now? I actually kinda dig the idea of having to EVA a kerbal over to the ship, perform a wake from cryo or whatever and then eva them both back, but I don't see any parts that would faciliatate this.

**Comment by brocksamson** [comment]
Wanted to update this with a few notes. 

I was able to send a kerbal (Jebediah) with a life support minipack to the distressed ship and that resolved the refuses to work issue.  There were some odd glitches, dunno if they are related or caused by another mod.  When I eva'd the stranded kerbal (Sidsen) he was in EVA and still listed in the crew list of the stranded vessel.  When I eva'd Jebediah out of the stranded vessel, he also remained in the stranded ship's crew list, but Sidsen vanished from the list.  When I had both kerbals back to my rescue ship I had 2 Jebediahs listed inside my rescue vehicle.  After completing the mission and recovering my return pod, everything seems fine.

Some more things I noticed in my multiple attempts to get this resolved:
The refused to work message is spamming multiple times always preceeded by a returns to work message.
When I uninstalled the mod to try to get around the issue that way, the stranded kerbal simply vanished from the distressed ship.
All changes within the settings.cfg were not applied to the stranded kerbal, I tried changing NoSupplyEffect & NoSupplyEffectVets to 0, and 3 without any effect.  I also tried changing the HomeWorldAltitude to 100,000 to see if that would do anything, but it did not.

Note that I didn't have a save game from before selecting the mission so each of these things was tried from a save  after accepting the resuce mission, but before I launched my rescue ship.

The stranded ship was a Deployable Emergency Shelter.
